spring-boot/autoconfigure/src/main/java/org/togglz/spring/boot/autoconfigure/TogglzProperties.java

```diff
@@ -380,6 +380,12 @@ public void setRegisterFeatureInterceptor(boolean registerFeatureInterceptor) {
 		}
 	}
 
+	/**
+	 * @deprecated Use Spring Boot 2 generic endpoint configuration instead
+	 *
+	 * @see <a href="https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-endpoints.html">Spring Boot 2 chapter on Endpoints</a>
+	 */
+	@Deprecated
```

Comment:
I wouldn't mark it as `@Deprecated` since the user had the choice to go for version 1 or 2 of Spring Boot. If he/she actively decided for 1, there is a reason and probably he/she is aware of that.

Reply:
Or do I miss something?

Reply:
I think you're right on this one, unless we also deprecate the Spring Boot 1 support, we should keep this the way it was. More thoughts on that: https://github.com/togglz/togglz/issues/268#issuecomment-388368382